### ISSUE TITLE
for loop not working for Dart Language

### ISSUE THREAD

**Issue by bivu029** [ApolloVM/apollovm_dart#15]
![for1](https://github.com/ApolloVM/apollovm_dart/assets/40115955/6f49b644-5e00-4d7f-acbe-722d49a296fe)

**Comment by gmpassos** [comment]
The code below works at:
https://apollovm.github.io/apollovm_web_example/www/

```dart

class Foo {

  void main(List<Object> args) {

    for ( var i = 0 ; i < 10 ; i += 1 ) {
      print('- $i');
    }

  }
  
}


```

I think that the issue is with `i++` or `++i`

**Comment by gmpassos** [comment]
Fixed on [apollovm: ^0.0.52](https://pub.dev/packages/apollovm/changelog)

Check it at:
https://apollovm.github.io/apollovm_web_example/www/

```dart

class Foo {

  void main(List<Object> args) {

    for ( var i = 0 ; i < 10 ; i++ ) {
      print('- $i');
    }

  }
  
}

```
